.github/workflows/build-plugin-archive.yml

```diff
@@ -203,7 +207,7 @@ jobs:
         run: echo "artifact=${{ steps.plugin-data.outputs.archive-name }}" >> $GITHUB_OUTPUT
 
       - name: Upload artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
@tyrann0us 

Thank you for pointing this out. I am not terribly familiar with the details involved here, but I actually needed this bump (also for the download action) in order to _download_ the artifact from the consuming workflow.

So this is not something I changed because it appeared to make sense - it's a requirement for this to work apparently.

However, I am not aware of any "production" workflows relying on this yet. I was experimenting on some things and needed this workflow to be usable full-circle. 
Now I will be experimenting on other stuff, so it would be okay if the `v4` bump happens later.

I just want you to be aware that merging this in `v3` might not make too much sense

Reply:
Thanks for the clarification. Makes sense.